### PR TITLE
Submit popup

### PR DESCRIPTION
**Pull request by lauradobson**
When testing, go incognito to make sure your cache is cleared

### CODE REVIEW

File: taworks/taform/static/taform/js/app-jquery.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 $(document).ready(function() {
+
```
Comment (j2kan) [inline taworks/taform/static/taform/js/app-jquery.js]: 3 extra lines removal and same below.

**Review by j2kan — COMMENTED** [review]
just few spaces removed and should the * be red?

File: taworks/taform/static/taform/js/app-jquery.js (hunk 1 above)
Comment (lauradobson) [reply taworks/taform/static/taform/js/app-jquery.js]: I think red symbolizes an error once it's been made (like how it happens after we submit and they don't fill it out) but @amyleblond @smwatts what do you think?

Comment (amyleblond) [reply taworks/taform/static/taform/js/app-jquery.js]: not standard for it to be reed

**Review by amyleblond — APPROVED** [review]
worked incognito :)

**Comment by j2kan** [comment]
red *

**Comment by j2kan** [comment]
@lauradobson @amyleblond @smwatts 
I think the * is removed from this PR?

https://github.com/UWaterlooMSCI342/F16-Team_2-TA_APPLICATION/blob/master/taform.php

^this was smuckers requirements for required before.

He wanted it to be red with ass tricks and the word (required) after.

I think just * is enough without the wold required.

Also what's on the current form right now, red ass tricks.

**Comment by j2kan** [comment]
Fields with an asterisk (*) must be completed

^ that should be red too.

Also can you add * to the course dropdown too?

File: taworks/taform/templates/taform/application.html [hunk 2]
```diff
@@ -57,7 +57,7 @@ <h1>Management Sciences TA Application Form</h1>
 <font class="font">{{error}}</font>
 <h2>Basic Information</h2>
 
-<p>Fields with an asterisk (*) must be completed. </p>
+<p id="p01">Fields with an asterisk (*) must be completed. </p id="p01">
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: I meant only the * itself should be red.

File: taworks/taform/templates/taform/application.html [hunk 3]
```diff
@@ -51,86 +51,89 @@ <h1>Management Sciences TA Application Form</h1>
...
-            <p>{{s_form.current_program.help_text}}</p>
-        </td>
-        <td>&emsp;{{s_form.current_program}}&emsp;</td>
-
-        <td>Department</td>
-        <td>
-            {{s_form.department.help_text}}
-        </td>
-        <td>&emsp;{{s_form.department}}&emsp;</td>
+<p id="p01">Fields with an asterisk (*) must be completed. </p id="p01">
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: i meant the * itself is red, the rest of the sentence should be black.

File: taworks/taform/templates/taform/application.html [hunk 4]
```diff
@@ -209,10 +214,14 @@ <h2>Confirmation</h2>
 
 <p><b>To submit your application you must agree to the above terms.</b></p>
 
-<p id="font">Please note you will not be able to make changes to your application once you submit.<p>
-<input type="submit" class="submitButton" id ="btn" value="Submit" disabled>
+<p id="p01">Please note you will not be able to make changes to your application once you submit.<p>
+<input name="confirm" type="submit" class="submitButton" id ="btn" value="Submit" disabled>
 </form>
 
+<div id="dialog">
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: add a class for red and name it something better than p01

File: taworks/taform/templates/taform/application.html [hunk 5]
```diff
@@ -51,86 +51,89 @@ <h1>Management Sciences TA Application Form</h1>
...
-            <p>{{s_form.current_program.help_text}}</p>
-        </td>
-        <td>&emsp;{{s_form.current_program}}&emsp;</td>
-
-        <td>Department</td>
-        <td>
-            {{s_form.department.help_text}}
-        </td>
-        <td>&emsp;{{s_form.department}}&emsp;</td>
+<p>Fields with an asterisk <span style="color:#FF0000">(*)</span> must be completed. </p>
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: Put css in style.css.

**Review by j2kan — CHANGES_REQUESTED** [review]
need to update style.css and better naming than p01

File: taworks/taform/templates/taform/application.html (hunk 4 above)
Comment (j2kan) [reply taworks/taform/templates/taform/application.html]: p01 was changed to successMessage @amyleblond

**Review by j2kan — APPROVED** [review]
lgtm

**Pull request merged**